requirements.txt

```diff
@@ -1,2 +1,2 @@
-GitPython==3.1.30
-pycryptodomex==3.9.8
+GitPython==3.1.41
```

Comment:
Why not the latest 3.1.42?

Reply:
good point, bumping both to their slightly higher available versions.